### ISSUE TITLE
[Reward] Add gold reward for reporting hackers

### ISSUE THREAD

**Issue by shirazmatas** [EarthMC/EarthMC.net#35]
Add gold (5-10 gold) for reporting exploits or hacks to staff to promote the discovery of hackers.

**Comment by Kyoris** [comment]
yES

**Comment by Pengun78** [comment]
You should get reward for reporting dupes, in total i have found 3 dupes in my server experience and have reported them all, but have gotten nothing in return while it saved the economy

**Comment by Aitonater** [comment]
And reporting alt's. There is very little incentive for people to report alts or not to alt. People dont even get warnings if they are open when they get caught. I don't think its a big enough deterrent. People are only open because they got caught not because they decided to come forward.

**Comment by Aitonater** [comment]
Alts also give players distinct advantages over others such as being able to hold multiple towns and produce more gold per day. Letting people off if they tell the truth once you catch them imo is bullshit, it doesnt stop them.

**Comment by FutureCW** [comment]
Hey I reported the SalC1 dupe and got nothing but minus like 50 claim chunks because of the rollback

**Comment by Toadally** [comment]
SlyPrince dislikes this

**Comment by anotherplayerofEMC** [comment]
This would be great !
Super idea, I hope it is implemented soon !

**Comment by DeadGlamour** [comment]
Or do that the reporter gets the gold of the hacker ;)

**Comment by maxifunseeker** [comment]
yeah, getting rewards would surely be an incentive for players to report other players who break the rules more actively

**Comment by maxifunseeker** [comment]
yeah, getting rewards would surely be an incentive for players to report other players who break the rules more actively

**Comment by DasCodes** [comment]
Same message.
